content/docs/fairing/fairing-overview.md

```diff
@@ -20,6 +20,8 @@ remotely with Kubeflow Fairing:
    a local notebook][gcp-local].
 *  Learn how to [train and deploy a model on GCP from a notebook hosted on
    Kubeflow][gcp-kubeflow].
+*  Learn how to [train and deploy a model on GCP from a notebook hosted on
+   AI Platform][gcp-ai-platform].
```

Comment:
For first usage on the page, give the full name of AI Platform.

Reply:
Done